tests/models/test_serialized_dag.py

```diff
@@ -206,6 +206,23 @@ def test_get_dag_dependencies_default_to_empty(self, dag_dependencies_fields):
         expected_dependencies = {dag_id: [] for dag_id in example_dags}
         assert SDM.get_dag_dependencies() == expected_dependencies
 
+    def test_order_of_dag_params_is_stable(self):
+        """
+        https://github.com/apache/airflow/issues/40154
```

Comment:
```suggestion
```

We can always find the why with git blame. I don't think this case is noteworthy enough to warrant a permanent link here.